### ISSUE TITLE
index out of range [1] when running getPageCount()

### ISSUE THREAD

**Issue by spaz926** [Lofter1/anyflip-downloader#1]
URL: https://anyflip.com/wltz/ylgh/

```
Preparing to download
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.getPageCount(0xc000170090)
        .../anyflip-downloader/main.go:130 +0x445
main.main()
        .../anyflip-downloader/main.go:30 +0x145
exit status 2
```

**Comment by Lofter1** [comment]
Hue, seems like Anyflip has at least 2 different config.js versions. I only ever encountered one of them until now. I think I need to adjust the regex for finding the Page Count a bit. Thanks for notifying me, will fix it!

**Comment by Lofter1** [comment]
It should now be fixed in the newest commit tagged v0.1.1. Closing Issue.

**Comment by spaz926** [comment]
Thanks for the fast update. Your fix works well.
